server.js

```diff
@@ -1,15 +1,15 @@
 // Core dependencies
+const fs = require('fs')
 const path = require('path')
 
 // NPM dependencies
+const bodyParser = require('body-parser')
 const dotenv = require('dotenv')
 const express = require('express')
-const flash = require('connect-flash')
 const nunjucks = require('nunjucks')
 const sessionInCookie = require('client-sessions')
 const sessionInMemory = require('express-session')
 const cookieParser = require('cookie-parser')
-const addNunjucksFiltersWithAppContext = require('./app/utils/filters-with-app-context')
```

Comment:
Need this too!

Reply:
This is a big one. This is something I think we should push upstream to the prototype kit, it's so useful.